kornia/feature/keynet.py

```diff
@@ -160,11 +160,11 @@ def forward(self, x: Tensor) -> Tensor:
             feats_i = self.feature_extractor(x)
             feats_i = F.interpolate(feats_i, size=(shape_im[2], shape_im[3]), mode='bilinear')
             feats.append(feats_i)
-        scores = self.last_conv(torch.cat(feats, dim=1))
+        scores = self.last_conv(concatenate(feats, dim=1))
```

Comment:
```suggestion
        scores = self.last_conv(concatenate(feats, 1))
```
numpy/jax axis, torch dim

Or maybe time to define our custom concatenate that accept both